spec/controllers/linked_data_terms_controller_spec.rb

```diff
@@ -495,12 +492,9 @@
         results = JSON.parse(response.body)
         expect(results).to be_kind_of Hash
         expect(results.keys).to match_array ['performance', 'results']
-        expect(results['performance'].keys).to match_array ['predicate_count', 'fetch_time_s', 'normalization_time_s',
-                                                            'fetched_bytes', 'normalized_bytes', 'fetch_bytes_per_s',
-                                                            'normalization_bytes_per_s', 'total_time_s']
+        expect(results['performance'].keys).to match_array ['fetch_time_s', 'normalization_time_s', 'fetched_bytes', 'normalized_bytes',
```

Comment:
predicate_count removed here.  It will be in the response_header added in PR #283.